charts/rd-judicial-data-load/Chart.yaml

```diff
@@ -3,12 +3,12 @@ appVersion: "0.1"
 description:  Data load from Judicial HR systems
 name: rd-judicial-data-load
 home: https://github.com/hmcts/rd-judicial-data-load
-version: 0.1.9
+version: 0.1.10
 maintainers:
```

Comment:
can we not just move to version 0.2 instead of 0.1.10?

Reply:
Makes sense. Have updated.